.github/ISSUE_TEMPLATE/eia_new_year.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Integrate New Year of EIA data
+about: Check-list for integrating a new year of EIA data
+title: ''
+labels: new-data
+assignees: ''
+
+---
+
+### EIA new year of data integration check-list:
+
+Review Annual Updates Docs: https://catalystcoop-pudl.readthedocs.io/en/dev/dev/annual_updates.html
+
+Replace references to `EIA_DATASET_NAME` with either `eia_860`, `eia923`, or `eia861` 
+depending on which EIA dataset you are updating.
+
+- [ ] Add the new Zenodo archive DOI to `pudl/workspace/datastore.py`.
+- [ ] Run the datastore script to download the new raw data: `pudl_datastore --dataset 
+EIA_DATASET_NAME`. The new raw data will appear in `pudl_input/eia923/<ZENODO_DOI>/...`
+- [ ] Update the mapping information in `pudl/package_data/EIA_DATASET_NAME` if 
+necessary:
+     - [ ] file maps (may need to add or remove _Early_Release suffix!).
+     - [ ] column maps (if there are new or chaned columns).
+     - [ ] page maps (if the tab order changes).
+     - [ ] skip footer (if there is new, non-data content at the end of the file).
+     - [ ] skip rows (the early release data tends to have an extra, skippable row at 
+     the top of the file. Either add or subtract that row depending on whether you are
+     adding early release or final data).
+- [ ] Launch dagit and refresh the code location (run in your terminal 
+`dagster-webserver -m pudl.etl` and then open 
+`http://127.0.0.1:3000/locations/pudl.etl/jobs/etl_full` in a browser)
+- [ ] Materialize the `raw_EIA_DATASET_NAME` asset group. Look out for warnings in the 
+logs about missing or extra columns. If they appear, check and update the `package_data` 
+accordingly.
+- [ ] Materialize the `_core_EIA_DATASET_NAME` asset group. Look out for warnings and 
+fix accordingly.
+- [ ] Materialize the `norm_eia` and then `denorm_eia` asset groups. You may see errors 
+related to encoding. Take a look at which column it's talking about and look at 
+`metadata/resources/eia.py` to see which encoder in `CODE_METADATA` to tweak.
+- [ ] Test table outputs in a Jupyter notebook to make sure expected dates appear.
```

Comment:
this feels oos for this pr for sure but it'd be good to have a page in our docs the `Developing with Dagster` section that you could point to here that would include the `defs.load_asset_value(AssetKey(asset_name))` setup for this.

Reply:
Yeah, would love to be able to point to an existing notebook that people could just load! But I think you're right it's OOS for this.